### PR TITLE
deleting excess boilerplate 

### PR DESCRIPTION
**Pull request by leestanley**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+
+function App() {
+  return (
+    <div className="App">
+        <h1>Hello world!</h1>
```
Comment (eminguyen) [inline src/index.js]: two space indents

File: prettier-config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    printWidth: 120,
```
Comment (eminguyen) [inline prettier-config.js]: two spaces

Comment (eminguyen) [reply prettier-config.js]: also prettier.config.js

File: prettier-config.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
```
Comment (eminguyen) [inline prettier-config.js]: what are you doing

Comment (leestanley) [reply prettier-config.js]: this hurts

**Comment by sumeet-bansal** [comment]
Testing something,

**Pull request merged**